src/EnergyPlus/DataSizing.cc

```diff
@@ -46,8 +46,10 @@
 // POSSIBILITY OF SUCH DAMAGE.
 
 // EnergyPlus Headers
+#include <EnergyPlus/DataEnvironment.hh>
 #include <EnergyPlus/DataPrecisionGlobals.hh>
 #include <EnergyPlus/DataSizing.hh>
+#include <EnergyPlus/SimAirServingZones.hh>
```

Comment:
@rraustad I'm guessing this is the culprit, because SimAirServingZones.cc includes DataSizing.hh. You could pass in `NumOfTimeStepInDay` as an argument. That would be cleaner anyway. And I don't see why `DataEnvironment` was added here.